FantasmoSDK/Classes/Network/FMRestClient.swift

```diff
@@ -18,33 +18,9 @@ struct FMRestClient {
     typealias RestResult = (Int?, Data?) -> Void
     typealias RestError = (Error) -> Void
     
-    // MARK: - internal methods
-    
-    /// Post a query to the CPS server
-    ///
-    /// - Parameters:
-    ///   - endpoint: The API endpoint to post to
-    ///   - parameters: Dictionary of form parameters
-    ///   - token: Optional API security token
-    ///   - completion: Completion closure
-    ///   - error: Error closure
-    static func post(_ endpoint: FMApiRouter.ApiEndpoint,
-                     parameters: [String : String],
-                     token: String?,
-                     completion: RestResult? = nil,
-                     error: RestError? = nil) {
-        
-        let request = Self.requestForEndpoint(endpoint, token: token)
-        log.info(String(describing: request.url), parameters: parameters)
-        
-        var data = Data()
-        data.appendParameters(parameters)
-        data.appendFinalBoundary()
-        
-        Self.post(data: data, with: request, completion: completion, error: error)
-    }
+    // MARK: - Internal methods
 
-    /// Post a query with an image to the CPS server
+    /// Post a query with parameters and an optional image to the CPS server
```

Comment:
This is unrelated to the task at hand.

Reply:
This is consequence of removing the redundant `post` method in `FMRestClient`. It was needed to refine the comment to other not removed method.

In my first plan was to implement in `FMApi` fully fledged `sendDeviceCharacteristics()` method and later to take from it obtaining params and remove if it will not be needed. `FMRestClient` related code was projected to quickly copy-past from other methods. And when investigating that intended to be copy-pasted code I noticed problem with code duplication in FMRestClient and decided to quickly fix it as not taking much time rather than create separate task for this. Improving error handling in `FMApi` was going to take significant time and I created separate task for it (https://linear.app/fantasmo/issue/MW-92/improve-error-handling-in-ios-sdk)

So this was the consequence of the quick improvement along the way which was highly likely to be forgotten otherwise.